src/clients/safeclientgateway.py

```diff
@@ -0,0 +1,39 @@
+import logging
```

Comment:
Filename must be camel case

Reply:
> Filename must be camel case

Same as here: https://github.com/gnosis/safe-config-service/pull/469#discussion_r833149938

Reply:
It doesn't need to be but I changed it [dfda6b1](https://github.com/gnosis/safe-config-service/pull/469/commits/dfda6b1f9e3fb6f4d75390c81c92a17c9c3043cd)